.changeset/sour-clocks-pay.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-common': minor
```

Comment:
Should be `patch` since there is no breaking change.

Reply:
I was thinking that since it is adding a backwards compatible feature and it extends the configuration schema it would be a minor but I can change it to patch if that's your convention.

Reply:
Since we're still in 0.x versions, a minor change is a breaking change and a patch change is non-breaking new features. :)